tensorboard/uploader/uploader_subcommand.py

```diff
@@ -462,12 +465,23 @@ def execute(self, server_info, channel):
         except KeyboardInterrupt:
             interrupted = True
         finally:
+            if self.one_shot and not uploader.tracker.has_data():
+                print(
+                    "Tensorboard was run in `one_shot` mode, but did not find "
+                    "any uploadable data in the specified logdir: %s\n"
+                    "An empty experiment was created. "
+                    "To delete the empty experiment execute the following\n\n"
+                    "    tensorboard dev delete --experiment_id=%s"
+                    % (self.logdir, uploader.experiment_id)
+                )
             end_message = "\n"
             if interrupted:
                 end_message += "Interrupted."
             else:
                 end_message += "Done."
```

Comment:
Thoughts on adding a newline before the Done line for the normal cases? It might be good to emphasize it, but that's just my opinion.